Protocol: Culturing A549 Cells in Serum Free Media with Stimulation by TNF-alpha for qPCR Analysis
============================================================================================================

Created by AG on 2016-03-01

# Protocol Notes

### Biohazard Safety

-    Be sure to work in a biohazard safety cabinet or fume hood at appropriate times. Wipe the biosafety cabinet before and after use with 70% ethanol or BacDown disinfectant. 
-    Use universal biosafety precautions, wear a barrier resistant lab coat, disposable gloves, and other appropriate PPE.
-    When disposing BAL fluids, add 10% bleach solution to the discarded fluid to disinfect the cells and fluid for at least 10 minutes ("10 for 10"), then discard and rinse down the drain. 
-    Dispose of trash in the designated "Biohazard" trash cans. Be sure to double bag the pipets before disposal. 
-    Qiazol/Triazol must be disposed of in the appropriate hazardous waste containers in the fume hood.


# Materials:

1. 15 mL polystyrene conical tubes
2. 50 mL polystyrene conical tubes
3. 5 mL polystyrene tubes with caps
4. 1.5 mL microcentrifuge tubes with caps
5. 0.2 mL PCR tubes, individual, located above thermocyclers
6. MicroAmp Fast Reaction Tube Strips, 8 tubes each, above working bench
5. Incubator (37C/5% CO2)
6. Growth Medium
   - Serum-Free Media (SFM), Location: refrigerator #6
   - Exosome-Free Media (EFM), Location: refrigerator #6
11. QIAzol or TRIzol Reagent (Equivalent for this)
   - Location: Middle shelf, in between the biosafety cabinets, or in 4C Fridge #6 & #7
12. TNF-alpha human at 5 ng/mcL
   - Location: Freezer #3, -20C, box #7
1. Chloroform, located under fume hood
2. Isopropyl alcohol, located under fume hood or within it
3. 100% Ethanol, flame proof cabinet next to fume hood, use and label your own bottle, do not use someone elses
3. DPEC water, your own, don't use someone elses, label & date your's
4. RNase-free water (alternative to DPEC)
5. RNA Wash Solution (75% EtOH / 25% DPEC)
   - 30 mL of EtOH plus 10 mL of DPEC
5. Sorval Biofuge stratos centrifuge
6. Heraeus 3332 rotor for Sorval Biofuge stratos
7. Nanodrop
1. BioRad T100 Thermocycler 
   - for cDNA synthesis
3. BioRad 200 mcL PCR tubes with domed lids
   - Above thermocycler, also in D42 (Slav's bench)
   - There are strips and plates, flat and domed lids
2. DNA Engine Gradient Cycler (alternative to BioRad)
3. Deoxyribonuclease I Amplification Grade
   - 10X DNase I Buffer as well
   - Invitrogen Cat # 18068-015
   - Location: Freezer #3, -20C
4. cDNA kit 
   - Applied Biosystems Cat# 4304134
   - 10X buffer, MgCl2 (25 mM), deoxyNTP mixture, random hexamers, RNase inhibitor, MultiScribe Reverse transcriptase
   - Location: Freezer #3, -20C
   - Extras locaated Freezer #5, -20C, third shelf, box on right
5. MicroAmp qPCR tubes with flat lids
   - Above benches and above qPCR machine
   - Come in strips of 8 and a plate of 96 wells

# Methods:

## Day 1:

1. Begin with 95% confluent A549 cells in appropriate sized container with EFM
2. Using 6-well culture plates
   1. Each well holds 2 mL of culture media
   2. Replace the exosome-free media (EFM) with serum-free media (SFM) in each well
   3. Incubate overnight at 37C/5% CO2

## Day 2:
   - Using 6-well culture plates 
   
1. Prepare TNF-alpha Stimulation Media
   - Desired final TNF-alpha concentrations of 15 ng/mL per well. The TNF-alpha is at 5 ng/mcL stock concentration.
   - For _each_ 6-well _plate_ combine to make a Master Mix:
      1. 12 mcL (10 ng/mL final); 18 mcL (15 ng/mL final); 24 mcL (20 ng/mL final) TNF-alpha stock, and 
      2. 6 mL serum-free media (SFM)
2. Add 2 mL of Stimulation Media to each _stimulation_ well
3. Add 2 mL of serum-free media (SFM) to each _control_ well
4. _Incubate_ at 37C/5% CO2 for _4 hours_ or overnight depending upon set-up
5. Remove supernatant for exosome isolation
6. Add 1 mL of TRIzol/QIAzol to each well and incubate for 5 minutes at RT
7. Gently wash about the TRIzol in each well
8. Remove the TRIzol mixture and place each well into a separate 5 mL polystyrene tube and cap
9. Place TRIzol tubes into the -80C freezer for later RNA extraction or proceed immediately to RNA extraction. 
10. *Alternatively* if processing RNA with a kit, then add only 700 mL of TRIzol or QIAzol to each well

## RNA Extraction - TRIzol/Manual Method
### Phase Separation
1. Incubate specimens for 5 minutes at RT if not already done (thaw if previously frozen)
2. Add 200 mcL of chloroform for each 1 mL of TRIzol Reagent 
3. Cap samples and *shake* vigorously for 15 seconds (do not vortex)
4. Incubate for 3 minutes at RT
5. Centrifuge samples at 12,000xg for 15 minutes at 4C 
   - Sorval Biofuge stratos centrifuge with Heraeus 3332 rotor (24 samples max using 1.5 mL tubes)
6. Mixture will separate into a lower red, phenol-chloroform phase, an interphase, and a colorless upper aqueous phase

### RNA Precipitation
1. Transfer the aqueous phase into a 1.5 mL centrifuge tube, care not to get any interface
   - cloudy portion is just around edges, so place pipet in center to remove aqueous phase
   - Set pipette to 300 mcL will get about 2 times full
2. Add 500 mcL of isopropyl alcohol (per 1 mL of TRIzol Reagent used initially) to precipitate RNA
3. Mix by gently inverting 10 times
4. Apparently can be frozen @ -20C at this stage

### RNA Wash
1. Incubate the samples for 10 minutes at RT with the isopropyl alcohol
2. Centrifuge at 12,000xg for 10 minutest at 10C
3. Discard supernatant
4. Wash RNA pellet *twice* with 1 mL of 75% EtOH / 25% DPEC (stock see above)
   - Briefly vortex or flick to tube to mix
   - Centrifuge at 12,000xg for 5 minutes at 4C
   - Place hinge up in the centrifuge always so know where pellete is
   - Remove residuals of wash with pipette carefully, trying not to disturb the pellet (don't remove all the fluid), only about 1000 mcL
   - Samples can be stored for up to one year at -20C in this mixture
6. Centrifuge again without adding any further wash solution at 12,000xg for 5 minutes at 4C
7. Remove any excess wash buffer with smaller 200 mcL pipette tip 
   - If not enough EtOH evaporates, this will prevent the RNA from going into solution moving forward
   - Bring it to nearly dry with the pipette, careful to not suck up the pellet
8. Air dry the samples by leaving the cap off on the bench (or in biosafety cabinet) for 5-10 minutes at RT
   - Mostly what is left here is EtOH, it should all evaporate within 10 minutes
   - Can do in culture hood if like
9. Resuspend in 20 mcL of DPEC (or RNAse-free water) by pipetting up and down and then heating for 10 minutes at 55-60C 
9. Use Nanodrop (RNA settings) to calculate RNA concentrations
   - Use 1.5 mcL of sample; print report out for notebook
   - OD260 nucleic acid; 260/280 like 1.8 or greater, 260/230 1.7 or greater
   - Lower numbers indicate protein, Qiazol, sugars or guanidine contamination
   - Absorbance does not correlate with RNA integrity
   - See separate protocol if needed
10. Store samples at -80C

## DNase Treatment of Total RNA
1. Turn on the heating block to high setting at 65C 
   - This is the usual setting for the one at the end of our bench
2. Thaw all reagents for this step and the cDNA step at this time
2. Make a Master Mix with the following volumes for each sample:
   - 2 mcL 10X DNAse I Buffer
   - 2 mcL DNase I, Amplification Grade, 1 U/mL concentration
3. Each sample will go into a 1.5 mL microcentrifuge tube 
4. Place 4 mcL of the Master Mix into each sample tube
5. Place calculated volume of each RNA sample into the appropriate sample tube
   - Use Slav's calculator spreadsheet to calculate volume of sample needed to yield 1 mcg of RNA for each sample
6. Place calculated volume of DPEC / RNase-free water into the appropriate sample tube
   - Use Slav's calculator (as above) to calculate the volume of water for each tube
7. Mix tubes by centrifugation briefly in benchtop Eppendorf Minispin
   - Do NOT vortex samples, DNAse will be damaged
8. Incubate tubes for 15 minutes at RT. 
   - Do not exceed 15 minutes as the RNA will begin to hydrolyze
9. Add 2 mcL of EDTA (25 mM concentration) to each tube to inactivate the DNase
10. Heat for 10 minutes at 65C in the heating block that is already warm
11. Samples are ready for cDNA preparation
   - Samples seem to be at 18 mcL (perhaps need to spin after the heatblock?)
 
## cDNA Synthesis

   - cDNA synthesis from total RNA is the first step in TaqMan Gene Expression Quantification assays.
   - Kit # 4304134 Applied Biosystems
   - All tubes are BioRad
      - Can use domed or flat tops, need to ensure proper setting on machine
   
1. Prepare reverse transcriptase reaction mix. 
   - Best to make a *Master Mix* of N + 1 samples.

| Component | Volume in mcL per Sample | Final Value per Sample |
|-----------|--------------------------|----------------------|
|25 mM MgCl2 | 22 | 5.5 mM|
|deoxyNTPs Mixture | 20 | 500 mcM (per dNTP)|
|DPEC (RNase-free water) | 18.5 mcL | 0.25-2.0 mcg |
|10X Buffer | 10 | 1X|
|Random Hexamers | 5 | 2.5 mcM |
|MultiScribe Reverse Transcriptase (50 U/mcL)| 2.5 | 1.25 U/mcL |
|RNase Inhibitor | 2 | 0.4 U/mcL |

2. Mix tubes by centrifugation briefly in benchtop Fisher Scientific Mini
3. Each sample goes into a 0.2 mL tubes in PCR strips (8 tubes each, BioRad) with domed cap.
   - Domed is standard. Can use flat top, just need to turn over the plate in the machine to match.
3. Place 80 mcL of the above *Master Mix* into each PCR tube.
4. Place 20 mcL (entire RNA sample after DNase step above) of each RNA sample into its own PCR
   - Total volume in each PCR tube will be 100 mcL (there about)
5. Mix tubes by centrifugation briefly in benchtop Fisher Scientific Mini
6. Run the thermal cycler
   - BioRad T100 Thermocycler
     - Select RT-CDNA program, 100 mcL, Run, close lid
   - DNA Engine Gradient Cycler
     - Run-> Proceed; RT-> Proceed; change to block 'B'-> Proceed; Tubes-> Proceed; Volume 100-> Proceed; Close top & tighten down with blue knob; Heated lid-> Proceed
   - Cycle should take about 48 minutes. 
7. cDNA can be used immediately for PCR amplification 
8. Storage
   - Transfer to a larger 1.5 mL centrifuge tube and label appropriately
   - Store in Freezer #3 -20C in appropriately labeled box
   
## qPCR

   - All tubes are MicroAmp fast reaction tubes
   - PDAR = Pre-Developed Assay Reagent
   - Below is for PDARs only (i.e., not primer/probe sets - see Slav protocol for this if needed)
   - Use between 10--100 ng/reaction of cDNA
   - Reagents should be kept in the dark!
   - Endogenous controls available are 18S, GAPDH, Beta Actin

1. Make a "Master Mix" of all reagents _without_ cDNA
   - will need a different Master Mix for each Target PDAR (i.e., IL-8 is one Master Mix, CXCL5 is another)
   - make enough for N + 1 as will need a Negative control (no cDNA)
   
| Component | Volume in mcL per Sample | Location | 
|-----------|--------------------------|------|
| 20 X Target Mix (i.e., IL-8, etc) | 1.0 | Freezer #5 -20C, see locator on outside |
| 20 X Control Mix (18S or GAPDH) | 1.0 | Freezer #1 -20C, aliquoted |
| RNase-free Water | 6.0 | DPEC is fine |
| TaqMan Gene Expression Master Mix | 10.0 | Fridge #1 4C |

2. Aliquot 18 mcL of Master Mix into each sample tube 
   - MicroAmp 100 mcL PCR tubes with flat MicroAmp tops
   - Can use a MicroAmp Optical 96-well plate instead (if more than half full)
   - If using srips, be sure to have empty wells so know which way putting in machine
   - Do _not_ write on the tubes or the plate!
   - Need to ensure appropriate sled is in the machine (kept in top drawer to right of the machine)
3. Add 2 mcL of sample cDNA into respective PCR tubes (excepting negative control)
4. If using the plate, cover the plate with MicroAmp Optical Adhesive Cover, seal throughout
5. Vortex gently
6. Spin at 3500 RPM for 10 seconds to eliminate bubbles using the Jouan CR 412 centrifuge
7. Set-up Machine Program
   - Open StepOnePlus Software V2.0
   - Choose Advanced Set-up
   - Enter Experiment Name (something useful with the date of the original experiment)
   - Select StepOne Instrument (96 wells)
   - Select Quantification: Comparative C<sub>t</sub>
   - Select Taqman Reagents
   - Select Standard, 2 hour run (you will be disappointed if you don't!)
   - Choose Plate Set Up
      - Add New Target
      - Enter target name and choose color
         - 18S = VIC
         - CXCL8 = FAM
      - Enter Sample names (need at least two names at this point, can add more later)
   - Choose Assign Targets and Samples
      - Select wells using and label them with the Target and Sample names
      - Select wells and choose the Endogenous control (may need to move the second column over to see this)
      - Select a control well (this is why you need at least two sample names)
   - Ensure ROX is selected as the Passive Reference
   - Choose Run Method and ensure conditions match those in table below
   - Choose Run
   - Will be prompted to save the experiment, please save to Aric's folder
   - Ensure the base rises up on the machine and the timer starts before walking away
   - Usually takes ~75 minutes
8. Analysis with software
   - Choose Analyze
   - Adjust the baseline, especially for highly amplified 18S
      - From the Graph Type menu dropdown, select Linear
      - Select Baseline checkbox to show the start and end cycle
      - Virfy that the baseline is set correctly: the end cycle should be set a few cycles before the cycle number where significant fluorescent signal is detected. 
   - Adjust threshold, should be set in the exponenetial phase of the amplification curve
      - From the Graph Type menu dropdown menu, select Log
      - From the Target dropdown menu, select 18S
      - Select Threshold checkbox to show threshold
      - Verify that threshold is set correctly
   - Click Analyze
   - Click Export
